airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java

```diff
@@ -121,7 +121,9 @@ private String extractAndCast(final ColumnId column, final AirbyteType airbyteTy
       // This is guaranteed to not be a Union, so we won't recurse infinitely
       final AirbyteType chosenType = u.chooseType();
       return extractAndCast(column, chosenType, forceSafeCast);
-    } else if (airbyteType instanceof Struct) {
+    }
```

Comment:
is this change necessary? I personally like the if/else chain (b/c then I can't accidentally put something important between two branches)